yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/HumanProfileAdapter.java

```diff
@@ -26,16 +23,14 @@
     private static final String ATTRIBUTE_DOB = "date_of_birth";
     private static final String ATTRIBUTE_AGE_OVER = "age_over:";
     private static final String ATTRIBUTE_AGE_UNDER = "age_under:";
-    private static final String ATTRIBUTE_GENDER = "gender";
+    public static final String ATTRIBUTE_GENDER = "gender";
```

Comment:
do you mind to group the public and private fields together pls?

Reply:
Maybe a better fix is to pull all the Attribute names out into one class, and make them all public...?

Reply:
This is also a good option